### PR TITLE
Update for Prometheus Community

### PR DESCRIPTION
**Pull request by SuperQ**
Migrate the codebase to the Prometheus Community
* Add licenses headers
* Add NOTICE file to record the copyright change.
* Remove non-standard goreleaser config.
* Remove non-standard .tool-versions config.
* Add promu and Prometheus common Makefile setup.
* Update Go module paths.

### CODE REVIEW

**Comment by thomaspeitz** [comment]
Small suggestions:
- security.md should point to prometheus-team@googlegroups.com
- Use prometheus-community as org name in LICENSE.md

**Comment by SuperQ** [comment]
There will probably be a few more things to clean up. Just wanted to get the basic stuff done to make the build work.

**Review by cristiangreco — APPROVED** [review]
Thanks @SuperQ! Will merge this today and then followup with other changes.

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Copyright 2018-2024 Invision AG
```
[the remark this answers is not in the document]
Comment (SuperQ) [reply NOTICE:1]: Sure, that makes sense. I wasn't fully aware of the history. Thanks for filling me in.

**Pull request merged**